src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -119,16 +119,17 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 semanticFacts.IsTypeContext(semanticModel, position, cancellationToken) ||
                 semanticFacts.IsTypeDeclarationContext(semanticModel, position, cancellationToken) ||
                 semanticFacts.IsNamespaceContext(semanticModel, position, cancellationToken) ||
+                semanticFacts.IsNamespaceDeclarationNameContext(semanticModel, position, cancellationToken) ||
                 semanticFacts.IsMemberDeclarationContext(semanticModel, position, cancellationToken) ||
                 semanticFacts.IsLabelContext(semanticModel, position, cancellationToken))
             {
-                return await GetSnippetCompletionItemsAsync(workspace, semanticModel, position, itemSpan, isPreProcessorContext: false, cancellationToken: cancellationToken).ConfigureAwait(false);
+                return await GetSnippetCompletionItemsAsync(workspace, semanticModel, itemSpan, isPreProcessorContext: false, cancellationToken: cancellationToken).ConfigureAwait(false);
             }
 
             return SpecializedCollections.EmptyEnumerable<CompletionItem>();
         }
 
-        private async Task<IEnumerable<CompletionItem>> GetSnippetCompletionItemsAsync(Workspace workspace, SemanticModel semanticModel, int position, TextSpan itemSpan, bool isPreProcessorContext, CancellationToken cancellationToken)
+        private async Task<IEnumerable<CompletionItem>> GetSnippetCompletionItemsAsync(Workspace workspace, SemanticModel semanticModel, TextSpan itemSpan, bool isPreProcessorContext, CancellationToken cancellationToken)
```

Comment:
Yes, unrelated. I noticed the param was unused as I was passing through. Should I revert it?